assets/styles/_print.scss

```diff
@@ -0,0 +1,84 @@
+@media print {
+  *,
+  *:before,
+  *:after {
+    background: transparent !important;
+    color: #000 !important;
+    /* Black prints faster */
+    box-shadow: none !important;
+    text-shadow: none !important;
+  }
+
+  [is='flynt-navigation-main'],
+  [is='flynt-navigation-burger'],
+  [is='flynt-navigation-footer'],
+  [is="flynt-block-cookie-notice"],
+  .button,
+  footer {
+    display: none !important;
+  }
+
+  a {
+    text-decoration: underline;
+    font-weight: bold;
```

Comment:
Why make links bold? I believe they should look like in the design, but black and underlined.

Reply:
okay,removed it.